src/AccessibilityInsights.Extensions.AzureDevOps/Properties/AssemblyInfo.cs

```diff
@@ -13,7 +13,7 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
 [assembly: AssemblyProduct("AccessibilityInsights.Extensions.AzureDevOps")]
-[assembly: AssemblyCopyright("Copyright �  2017")]
+[assembly: AssemblyCopyright("Copyright �  2017")]
```

Comment:
This new copyright character isn't rendering in the diff for me, is this an intended change?

Reply:
Nice catch! fixed